### DISCUSSION TITLE
Oikofixiin tekstin tavuttaja

### DISCUSSION THREAD

**Discussion by erkkaopena** [voikko/discussion#9] [General]
Olisiko mahdollista saada Oikofixiin myös tekstin tavuttamisosio? Vanha voikko-tavuttaja oli todella iso hyöty alakoulun opettajien työssä ja nykyään vanha sivusto ei enää toimi eikä tekstejä saa tavutettua. Tekstin tavuttaminen itse näpytellen on todella työlästä ja tavuttajan avulla pystyi muuttamaan minkä tahansa tekstin aloitteleville lukijoille sopivammaksi.

**Comment by ahomansikka** [comment]
Linuxissa:

fmt -1 teksti.txt | voikkohyphenate | fmt

**Reply by hatapitk** [reply]
Hei! Vanha tavuttaja on palautettu syksyllä, löytyy osoitteesta https://joukahainen.puimula.org/hyphenate
